### PR TITLE
Added Loading Spinner to pages

### PR DESCRIPTION
**Pull request by jj7lin**
Added loading spinner to the following pages:
CSS:
- edit profile page/settings page
- view profile page
- matches page 
- swiping page

Other:
- Added conditional rendering to the js files for the respective pages
- created loading spinner js file
- created loading spinner css file that can be called on every page

i would like to thank evette 4 her help

image

### CODE REVIEW

File: src/Matches/Matches.js [hunk 1]
```diff
@@ -2,88 +2,91 @@ import React from "react";
...
+    };
+    this.selectUser = this.selectUser.bind(this);
+  }
+  selectUser(event) {
+    const userSelected = event.target.value.split("|");
+    const myRequest = new Request("http://127.0.0.1:5000/conversationId", {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify({
+        userId: this.state.userId,
```
Comment (echanlee) [inline src/Matches/Matches.js:35]: why did you remove the quotation marks? does this still work properly without them

File: src/Profile/LoadingSpinner.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import "./LoadingSpinner.css";
+
+class LoadingSpinner extends React.Component {
+  render() {
+    return <div id="Loading"></div>;
```
Comment (echanlee) [inline src/Profile/LoadingSpinner.js]: I do not think this actually creates a loading spinner. I think you still need to import the css.

File: src/Matches/Matches.js [hunk 3]
```diff
@@ -97,128 +100,170 @@ class Matches extends React.Component {
...
+      );
+    } else {
+      matchingSection = <h2></h2>;
+    }
     return (
-      <div id="Matches-section">
-        {matchingSection}
+      <div className="matchingComponent">
+        {loading ? (
+          <LoadingSpinner /> && <div className="Loading"></div>
```
Comment (echanlee) [inline src/Matches/Matches.js]: What is this line supposed to do?

File: src/Profile/LoadingSpinner.js (hunk 2 above)
Comment (echanlee) [reply src/Profile/LoadingSpinner.js]: You should be able to place the `<\LoadingSpinner >` component to enable the loading spinner and I'm pretty sure it won't work right now.

**Review by echanlee — CHANGES_REQUESTED** [review]
When you make changes the spacing of most files change. Can you try to make sure you don't change all the spacing so it's easier to see what code you changed.

File: src/Profile/LoadingSpinner.js (hunk 2 above)
Comment (jj7lin) [reply src/Profile/LoadingSpinner.js]: it works for me? css is already imported on line 2

File: src/Matches/Matches.js (hunk 1 above)
Comment (jj7lin) [reply src/Matches/Matches.js:35]: yeah
